### PR TITLE
CUBA 7.0 migration

### PR DESCRIPTION
**Pull request by zaharchenko-evgeny**
Hi Mario, I'm working on CUBA 7.0 migration for Admin tools CUBA component. It is required to update runtime-diagnose component too.
Please review my changes for compatibility with CUBA 7.0.

### CODE REVIEW

File: modules/core/db/update/hsql/18/181119-1-createDiagnoseExecutionLog.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+create table DDCRD_DIAGNOSE_EXECUTION_LOG (
```
Comment (mariodavid) [inline modules/core/db/update/hsql/18/181119-1-createDiagnoseExecutionLog.sql:1]: why is a table created here?

File: modules/web/src/de/diedavids/cuba/runtimediagnose/web/screens/console/ConsoleWindow.groovy [hunk 2]
```diff
@@ -1,78 +1,130 @@
...
-abstract class ConsoleWindow extends AbstractWindow {
+class ConsoleWindow extends AbstractConsoleWindow {
 
-    public static final int SPLIT_POSITION_CENTER = 50
+    @Inject
+    DbDiagnoseService dbDiagnoseService
 
     @Inject
-    SourceCodeEditor console
+    ComponentsFactory componentsFactory
```
Comment (mariodavid) [inline modules/web/src/de/diedavids/cuba/runtimediagnose/web/screens/console/ConsoleWindow.groovy:26]: are you still using it - or switched to `UiComponents` bean?

**Review by mariodavid — COMMENTED** [review]
hi, thanks for the effort taken. Will run it locally and see how it goes. I added some comments on stuff that I noticed while scrolling over...

File: modules/core/db/update/hsql/18/181119-1-createDiagnoseExecutionLog.sql (hunk 1 above)
Comment (zaharchenko-evgeny) [reply modules/core/db/update/hsql/18/181119-1-createDiagnoseExecutionLog.sql:1]: Hi Mario,
looks like studio automatically created this update script

File: modules/web/src/de/diedavids/cuba/runtimediagnose/web/screens/console/ConsoleWindow.groovy (hunk 2 above)
Comment (zaharchenko-evgeny) [reply modules/web/src/de/diedavids/cuba/runtimediagnose/web/screens/console/ConsoleWindow.groovy:26]: I think it should be switched to `UiComponents` bean, `ComponentsFactory` is deprecated in CUBA 7.0

**Comment by mariodavid** [comment]
hi,

I continued the work here: https://github.com/mariodavid/cuba-component-runtime-diagnose/pull/27

Thanks for your contribution 👍

**Pull request closed without merging**